### PR TITLE
Migrate csharp scopes

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [/] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [/] I have not broken the cheatsheet

### CODE REVIEW

**Comment by AndreasArvidsson** [comment]
@pokey Another one bites the dust!

**Review by pokey — COMMENTED** [review]
missing value iteration scope. otherwise looks good

**Comment by AndreasArvidsson** [comment]
> missing value iteration scope. otherwise looks good

`value.mapPair.iteration`?

**Comment by pokey** [comment]
Yeah that and value for assignment if that's supported and value for default arg if that's supported

**Comment by AndreasArvidsson** [comment]
Done

File: data/fixtures/scopes/csharp/value.variable.scope
Comment (pokey) [inline data/fixtures/scopes/csharp/value.variable.scope:1]: need iteration scope for this one too right?

Comment (pokey) [reply data/fixtures/scopes/csharp/value.variable.scope:1]: "name" prob wants the same one. Just a block of statements

Comment (AndreasArvidsson) [reply data/fixtures/scopes/csharp/value.variable.scope:1]: Added 
https://github.com/cursorless-dev/cursorless/blob/890ad1bd463a348e7437769ace532fa7f7ccbdd6/queries/csharp.scm#L283-L290

Comment (pokey) [reply data/fixtures/scopes/csharp/value.variable.scope:1]: good enough

**Review by pokey — APPROVED** [review]

**Pull request merged**